Mathlib/GroupTheory/GroupExtension/Defs.lean

```diff
@@ -22,14 +22,14 @@ such as splittings and equivalences.
 
 ```text
 For multiplicative groups:
-      ↗︎ E  ↘
+      ↗ E  ↘
 1 → N   ↓    G → 1
-      ↘︎ E' ↗︎️
+      ↘ E' ↗
 
 For additive groups:
-      ↗︎ E  ↘
+      ↗ E  ↘
 0 → N   ↓    G → 0
-      ↘︎ E' ↗︎️
```

Comment:
I think these are genuinely intended to be non-breaking; the right solution is to put this in a code block.

Reply:
Sorry these are about the text-variant-selector of the arrows characters, i.e. there were invisible `\uFE0E` following the diagonal arrows to prevent them form being displayed as emojis.

There are no spaces changed here.

If they are supposed to have the variant-selector (which is a fair choice) then, they can be added to the list of text-symbols defined here and the selectors will be added back automatically by `lake exe lint-style --fix`